code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -318,6 +318,14 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 			light(SPAN_WARNING("[user] swings their [W], barely missing themselves. They light their [name] in the process."))
 	return
 
+/obj/item/clothing/mask/smokable/cigarette/catch_fire()
```

Comment:
Uh, isn't this already covered by fire_act? Or is that just for hotspot exposure or whatever it's called.

Reply:
![image](https://user-images.githubusercontent.com/22774890/101513761-36566f80-3985-11eb-8ecf-93f233ca38d7.png)
hotspot_expose has 4 matches but none of them cause it to light